content/kubermatic/master/architecture/concepts/presets/_index.en.md

```diff
@@ -55,15 +76,20 @@ spec:
     tenant:
     domain: DEFAULT
     floatingIpPool: ext-net
+    network:
+    securityGroups:
+    routerID:
+    subnetID:  
   packet:
     apiKey:
     projectId:
+    billingCycle:  
   vsphere:
     username:
     password:
     datastore:
     datastoreCluster:
-    datacenter:
+    vmNetName:
   kubevirt:
     kubeconfig:
 ```
```

Comment:
I also see that all of the providers embed PresetProvider which has 

```
type PresetProvider struct {
	Enabled    *bool  `json:"enabled,omitempty"`
	Datacenter string `json:"datacenter,omitempty"`
}
```
do we want to show this in the yaml as well?

Reply:
no, don't need to show this